<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TuringInterface</class>
 <widget class="QDockWidget" name="TuringInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Turing Machine Interface</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Please enter the following parameters:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Tape String</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tapeString">
        <property name="text">
         <string>aab#aab </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Qstart</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="qStart">
        <property name="text">
         <string>q0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Qaccept</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="qAccept">
        <property name="text">
         <string>qacc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Console:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="consoleDisplay">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="horizontalGroupBox">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Push</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="executeButton">
           <property name="text">
            <string>Execute</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
